_i18n/en.yml

```diff
@@ -153,6 +153,7 @@ merchants:
   centrexchangesp: If you prefer to use centralized exchanges, here is a list of renowned CEXes and swappers. Many more exchanges support Monero, we list here only a few reputable ones.
   cexp: Centralized exchanges that offer exchanging Monero for national currencies and cryptocurrencies.
   swappersp: Swappers let users exchange XMR for other cryptocurrencies.
+  aggregatorsp: Aggregators compare all exchanges at once and find cheapest, fastest or best rated exchanges.
```

Comment:
Is it all "exchanges" or all "swappers"?

Reply:
swappers*